docs/docs/references/metrics.md

```diff
@@ -65,3 +69,22 @@ metrics:
     type: count_distinct
     sql: "IF(${subscriptions.is_active}, ${user_id}, NULL)"
 ```
+
+## Using custom SQL in non-aggregate metrics
```

Comment:
I wonder if non-aggregate is the right name for these derived metrics? It's more like you can just write whatever you want as long as it results in an aggregated metric (so it kinda is aggregated?).

Maybe "custom aggregations" "derived aggregations"

or auto-aggregate vs. manual-aggregate

Reply:
I get non-aggregate means it doesn't aggregate **for you** but this metric (like all metrics) actually will be an aggregate value. So a bit confusing

Reply:
Not sure if I agree with 'manual-aggregate'. Can't we have a metric that isn't an aggregate ? For example for the boolean type, checking if a combination of values are present.  
In that case we might need something like `aggregate` vs `freestyle`

Reply:
I love freestyle haha

Reply:
Ok, let's stick with Looker names. Since they are well known